docs/sources/next/using-k6/javascript-typescript-compatibility-mode.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: JavaScript compatibility mode
-menuTitle: JavaScript mode
-excerpt: 'k6 supports running test scripts with different ECMAScript compatibility modes using --compatibility-mode'
-_build:
-  list: false
+aliases:
+  - ./javascript-compatibility-mode/ # /docs/k6/<K6_VERSION>/using-k6/javascript-compatibility-mode/
+title: JavaScript and TypeScript compatibility mode
+menuTitle: JavaScript and TypeScript mode
```

Comment:
@joanlopez @szkiba what do you think of changing the title to mention TypeScript so it's easier for users to find?

Reply:
SGTM!

Reply:
@heitortsergent 
> what do you think of changing the title to mention TypeScript so it's easier for users to find?

agree

@ppcano 
> I am not sure if naming new features as experimental is always the best choice, as it may prevent adoption

I agree, originally I did not use the experimental prefix, I added it at the request of the core team.

> What do you think about changing the mode to typescript (instead of experimental_enhanced)?

The idea behind it was that esbuild preprocessing is not only suitable for typescript transpilation, but also has added value for javascript.

Reply:
Agree that the name can be a bit confusing, but I think this should be a separate discussion in the k6 repo. 🙏 We can update the docs if we change the name.